src/main/steps/applicant1/full-name/content.test.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable jest/expect-expect */
+import { FormContent, FormFields } from '../../../app/form/Form';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { cy, en, generateContent } from './content';
+
+const CY = 'cy';
+const EN = 'en';
+const cyContent = cy();
+const enContent = en();
+
+const langAssertions = (language, content) => {
+  const generatedContent = generateContent({ language, userCase: {} } as CommonContent);
+  const { section, title, fullName, errors } = content;
+
+  expect(generatedContent.section).toEqual(section);
```

Comment:
This is good approach however we won't be able to catch the unwanted changes in content files by this method
for example if someone changes `title` from `Primary applicant` to `Primary aplicant`